napari/layers/base/base.py

```diff
@@ -250,7 +255,7 @@ def __init__(
         self._blending = Blending(blending)
         self._visible = visible
         self._freeze = False
-        self._status = 'Ready'
+        self._status = ''
```

Comment:
We could get the translate `DEFAULT_STATUS_TEXT` from `napari.components._viewer_constants`, but I suspect it's not too important here. Can add if desired, though unsure if that will cause any circular imports - probably not and can fix that if needed.